### DISCUSSION TITLE
How to modify the URL in the runtime window？

### DISCUSSION THREAD

**Discussion by InfernalAzazel** [tauri-apps/tauri#6327] [Q&A] [answered]
let main_window =app.get_window("main").unwrap();

**Comment by FabianLars** [comment, marked as answer]
something like this
```rs
let main_window =app.get_window("main").unwrap();
let _ = main_window.eval("window.location.replace('https://google.com')");
```
At some point there will exist an actual API for that.

**Reply by InfernalAzazel** [reply]
I tried your code without any effect

**Reply by InfernalAzazel** [reply]
Thank you very much for your help. I have solved it. The following is the correct code.

```rust
let main_window =app.get_window("main").unwrap();
main_window.eval("window.location.replace('https://google.com')").unwrap();
```

**Reply by FabianLars** [reply]
whoops, typed it on my phone without thinking, sorry! Edited my answer in case someone else comes here and doesn't check the comments.
